api/services/mapping/models/CaveModel.js

```diff
@@ -3,21 +3,4 @@ module.exports = {
   '@context': 'https://ontology.uis-speleo.org/grottocenter.org_context.jsonld',
   '@id': undefined,
   '@type': 'https://ontology.uis-speleo.org/ontology/#UndergroundCavity',
-  author: undefined,
-  dateInscription: undefined,
-  dateReviewed: undefined,
-  depth: undefined,
-  descriptions: [],
-  documents: [],
-  entrances: [],
-  histories: [],
-  id: undefined,
-  isDeleted: undefined,
-  isDiving: undefined,
-  length: undefined,
-  massifs: [],
-  name: undefined,
-  names: [],
-  reviewer: undefined,
-  temperature: undefined,
```

Comment:
I understand we should do something about these models which are not very useful (because now it's pretty clear in the converters, what are the attributes used) but still : why deleting these lines ?

Reply:
Because, as you said theses properties are useless now and will be overwritten in the converter. To improve this maybe I could either delete the cave model and put the fixed properties directly in the converter or put them back for a future global models refactor ?

Reply:
Ok, let's remove the undefined fields.

About the JSON-LD values, I would keep them in the models to clearly see which properties are set by default without having to dive into the converters' code. And maybe rename the folder "defaultModels" instead of "models" ?